exp-android-sdk/src/main/java/com/scala/exp/android/sdk/model/Device.java

```diff
@@ -42,13 +42,13 @@ public void setExperience(Experience experience) {
         this.experience = experience;
     }
 
-    public static ExpObservable<Device> getCurrentDevice(){
+    public static ExpObservable<?> getCurrentDevice(){
```

Comment:
something like this return new ExpObservable<Device>(Observable.<Device>just(null));